src/main/java/ladder/domain/Ladder.java

```diff
@@ -38,17 +38,20 @@ public List<Boolean> hasRightConnections(int heightPosition) {
     }
 
     public int climbUser(User user) {
-        int position = users.findUser(user);
-        for (int i = 0; i < height(); i++) {
-            position += ladderRows.get(i).move(position);
-        }
-        return position;
+        return IntStream.range(0, height())
+                .reduce(users.findUser(user), (acc, i) -> acc + ladderRows.get(i).move(acc));
```

Comment:
IntStream을 자주 사용하셔서 코멘트 드리기 조심스러운데요
ladderRows의 타입을 활용해 Stream API를 활용하는 것도 고려해 보시면 좋을 것 같아요
```suggestion
        return ladderRows.stream()
                .reduce(users.findUser(user),
                        (acc, ladderRow) -> acc + ladderRow.move(acc),
                        Integer::sum);
```

Reply:
오 감사합니다!